script/tool/lib/src/drive_examples_command.dart

```diff
@@ -376,19 +380,27 @@ class DriveExamplesCommand extends PackageLoopingCommand {
     File driver,
     List<File> targets, {
     required List<String> deviceFlags,
+    required String exampleName,
   }) async {
     final List<File> failures = <File>[];
 
     final String enableExperiment = getStringArg(kEnableExperiment);
+    final Directory? logsDirectory = ciLogsDirectory(platform, driver.fileSystem);
 
     for (final File target in targets) {
+      Directory? screenshotDirectory;
+      if (logsDirectory != null) {
+        screenshotDirectory = logsDirectory.childDirectory('$exampleName-drive');
+      }
```

Comment:
`Directory? screenshotDirectory = logsDirectory?.childDirectory(...);`

Reply:
Oops, I copied code that was written before null safety and clumsily migrated (likely by me).